mover-syncthing/Dockerfile

```diff
@@ -0,0 +1,85 @@
+# Build the Syncthing image from Golang
+FROM golang:1.17 AS syncthing-builder
+
+# source directory
+RUN mkdir -p /workspace
+WORKDIR /workspace
+
+# latest release as of writing this
+ARG SYNCTHING_VERSION="v1.18.4"
+ARG SYNCTHING_GIT_HASH="951b0589522e13e14e8b40d33ea66de1e69c8b4e"
+
+# clone & cd
+RUN git clone \
+  --depth 1 \
+  --branch=${SYNCTHING_VERSION} \
+  https://github.com/syncthing/syncthing.git
+WORKDIR /workspace/syncthing
+
+# Make sure we have the correct Syncthing release
+RUN /bin/bash -c "[[ $(git rev-list -n 1 HEAD) == ${SYNCTHING_GIT_HASH} ]]"
+# Create the binary for syncthing 
+RUN go run build.go
```

Comment:
I think we want to disable autoupgrade. I think that amounts to adding `-no-upgrade` to this command